src/utils/logger.js

```diff
@@ -0,0 +1,15 @@
+const logger = require('pino');
+
+module.exports = logger(
+  {
+    transport: {
+      target: 'pino-pretty',
+      options: {
+        colorize: true,
+        translateTime: 'SYS:dd-mm-yyyy HH:MM:ss',
+        ignore: 'pid,hostname',
+      },
+    },
+  },
+  logger.destination('../voluntera.log')
+);
```

Comment:
`pino-prettier` default destination is STDOUT. I believe you need to use 2 transports to implement this. One for info logs which can use pino-prettier and one for errors. Errors should be written inside a file.

You can achieve this by using multiple targets with different levels:
```
const pino = require('pino')
const transport = pino.transport({
  targets: [{
    level: 'info',
    target: 'pino-pretty' // must be installed separately
  }, {
    level: 'error',
    target: 'pino/file',
    options: { destination: '/path/to/store/logs' }
  }]
})
pino(transport)
```

Reply:
solved, thank you eniz @ZuluNovember